Car.cpp

```diff
@@ -0,0 +1,44 @@
+#include "Car.hpp"
+
+#include <iostream>
+
+#include "Exceptions.hpp"
+
+Car::~Car() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake() {
+    std::cout << __FUNCTION__ << std::endl;
+    speed_ = 0;
+}
+
+void Car::accelerate(int speed) {
+    std::cout << __FUNCTION__ << std::endl;
+    if (speed >= 0 && speed < 300) {
```

Comment:
i suggest you to avoid magic numbers. Just make them constexpr or const values. For example minSpeed, maxSpeed

Reply:
Thanks, I will change it :)